spec/controllers/bills_controller_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+
+describe BillsController do
+
+  describe "GET #index" do
+    let!(:bill) { FactoryGirl.create(:bill) }
```

Comment:
Ah, I will change `let!` to `let` where it is not required.

Yup, I kept running into problems with putting `let` into a before statement outside of the `describe` scope, but I didn't realize I could use `@bill` instead of `let(:bill)`.

Pushing these changes soon!